source/system/Numbers.ooc

```diff
@@ -100,7 +100,7 @@ LDouble: cover from long double {
 		result
 	}
 	in: func (range: Range) -> Bool {
-		this >= range min && this < range max
+		this as Int >= range min && this as Int < range max
```

Comment:
> Since range min and range max is Int this should be same as previously.
I assume that previously, `range min/range max` was converted to `This`, while it now truncates `this` before comparison. 

> However what does "as Int" mean
cast to int

I felt it was inreasonable to expect `2.1 in (1 .. 2)` to be invalid. Coming to think of it, I would expect this to succeed so I wil update it :D